packages/app-headless-cms/src/admin/plugins/index.ts

```diff
@@ -3,16 +3,17 @@ import menus from "./menus";
 import fields from "./fields";
 import validators from "./validators";
 import icons from "./icons";
-
+import install from "./install";
 // import header from "./contentDetails/header";
 import revisionContent from "./contentDetails/revisionContent";
 import previewContent from "./contentDetails/contentForm";
 import header from "./contentDetails/header";
-import pageRevisions from "./contentDetails/pageRevisions";
+// import pageRevisions from "./contentDetails/pageRevisions";
```

Comment:
Remove this line entirely.

Reply:
Should I also remove the other commented out code?

Reply:
Yeah, remove it, and I'll check these files that are commented. They are part of the content model form, which I'm currently working on. 👍

Reply:
:cool: 
Thanks @doitadrian